test/test_callframe.py

```diff
@@ -145,6 +147,13 @@ def test_describe_CFI_instructions(self):
             (   '  DW_CFA_def_cfa: r7 (edi) ofs 2\n' +
                 '  DW_CFA_expression: r2 (edx) (DW_OP_addr: 201; DW_OP_deref; DW_OP_deref)\n'))
 
+    def test_CFIEntry_get_decoded(self):
+        test_dir = join('test', 'testfiles_for_unittests')
+        with open(join(test_dir, 'simple_mipsel.elf'), 'rb') as f:
+            elf = ELFFile(f)
+            di = elf.get_dwarf_info()
+            entries = di.CFI_entries()
+            entries[0].get_decoded()
```

Comment:
Can you extend this a bit to extract some useful information from get_decoded and test it for some expected value?